packages/render-fidelity-tools/test/renderers/vray/README.md

```diff
@@ -0,0 +1,9 @@
+## Requirements
+
+- [V-Ray App SDK](https://www.chaos.com/vray/application-sdk)
+- A V-Ray license
```

Comment:
Thank you! Considering none of the maintainers of model-viewer have a V-Ray license, will someone commit to updating these renders occasionally? In particular, I'd like to know who to mention when a PR comes in that adds new sample models. @bsdorra has been kind enough to keep up with this for Stellar.

Reply:
Perhaps @bhouston?

Reply:
I think this is ready to merge if @bhouston agrees - I just want to make sure someone can actually update these.

Reply:
Let's do it!  This is an amazing push forward for the render fidelity tests!

Reply:
@bhouston I still need you to agree that you're on the hook and able to update these Vray renders going forward. Can I depend on you?

Reply:
Yes, I can commit to that for the foreseeable future - I have it working on my computer as well.  I have also let Mr V-Ray himself, @vkoylazov, know that we are doing this and he is supportive.

Reply:
Excellent, thank you!